tools/code_format/check_format.py

```diff
@@ -925,12 +937,10 @@ def checkFormat(file_path):
     if try_to_fix:
       error_messages += fixSourcePath(file_path)
     error_messages += checkSourcePath(file_path)
-
   if error_messages:
     return ["From %s" % file_path] + error_messages
   return error_messages
 
-
```

Comment:
Please don't do random changes like this. CI is still failing `fix_format`, can you take a look? Thanks.

Reply:
My bad :( 
don't know why these types of errors is arrives. let me correct my local env. 
strange is all format checks are passing in my local env